src/test_l1_plat_power.c

```diff
@@ -690,7 +692,7 @@ void test_l1_plat_power_negative_PLAT_API_GetWakeupSrc (void)
 {
```

Comment:
Line 498 : For PLAT_API_SetWakeupSrc loop for the wakeup src available from yaml file

Reply:
Done